### ISSUE TITLE
gnuplot Qt missing library

### ISSUE THREAD

**Issue by lczech** [popgenmethods/smcpp#166]
In the command `cv`, I get the error message

    /usr/bin/gnuplot: error while loading shared libraries: libQt5Core.so.5: 
    cannot open shared object file: No such file or directory

although Qt5 is loaded. The `cv` cmmand still seems to continue, and I have no idea why gnuplot is called within that command, but I figured I should report that here. Maybe it is not important - in that case, feel free to close the issue immediately ;-)

**Comment by terhorst** [comment]
It prints a text plot of the current size history estimate at each iteration. 

Assuming you're running Docker, it seems to be this bug: votca/buildenv#22. A possible fix is votca/buildenv#24. I'll try that and see what happens.

**Comment by lczech** [comment]
Yes, I'm running Docker via Snakemake. Your fix worked - I get an ASCII drawing of the model now:

```
      1e+06 +-------------------------------------------------------------+   
            |+           +                                               +|   
            |+                                             Pop. 1 *******+|   
            |+                                                           +|   
            |                                                             |   
            |+                                                           +|   
            |+                                                           +|   
            |                                                             |   
 N_e        |+                                                           +|   
            |                                                             |   
            |+                                                           +|   
            |                             B****************               |   
            |                              *              *               |   
            |                             *                *              |   
            |******************************                B**************|   
     100000 +-------------------------------------------------------------+
```

Awesome! Thank you for the quick fix!

Still, after that, the processing fails due to https://github.com/popgenmethods/smcpp/issues/165 :-(
